Archive/PiBase/Theorems.lean

```diff
@@ -0,0 +1,48 @@
+import Mathlib.Topology.Basic
+import Mathlib.Data.Set.Basic
+import Mathlib.Order.Filter.Basic
+import Archive.PiBase.Properties
+open Topology Set Filter
+
+universe u
+variable (X : Type u) [TopologicalSpace X]
+
+namespace πBase
+
+theorem T119 : P2 X → P1 X := by
```

Comment:
This is a general comment: I think putting [P2 X] before the “:” is more Mathlib style, and shortens the proof.

Reply:
Would this work for theorems of the form ~P → Q? I suppose we could have notPN defined to be ~PN for each property.

Reply:
I just listed them as hypotheses `h : \neg P` for simplicity. You can also define notPN indeed, I'm not sure which approach is preferable.

Reply:
Oh, sure, that would work just fine, wouldn't it. 😅 Makes sense to me.